regolith/schemas.py

```diff
@@ -3620,6 +3621,172 @@
+                                           "not needed for paper submissions",
+                            "required": False,
+                            "type": "list"},
+                "notes": {"description": "deliverable note",
+                          "required": False,
+                          "type": "list"},
+                "status": {"description": "current state of deliverable",
+                           "required": False,
+                           "type": "string",
+                           "eallowed": PROJECTA_STATUS},
+            }
+        },
+        "description": {
+            "description": "explanation of projecta",
+            "required": False,
+            "type": "string"
+        },
+        "end_date": {
+            "description": "projecta end date, yyyy-mm-dd. "
+                           "Not necessarily the same as deliverable due date",
+            "required": False,
+            "anyof_type": ["date", "string"]
+        },
+        "grants":{
+            "description": "grant id funding the project",
+            "required": False,
+            "type": "string"
+        },
+        "group_members": {
+            "description": "group member id's working on this project",
+            "required": False,
+            "type": "list"
+        },
+        "kickoff":{
+            "description": "details the projecta kickoff meeting",
+            "required": False,
+            "type": "dict",
+            "schema": {
+                "date": {"description": "kickoff meeting date, yyyy-mm-dd.",
+                         "required": False,
+                         "anyof_type": ["date", "string"]},
+                "due_date": {"description": "kickoff meeting by date, yyyy-mm-dd.",
+                             "required": False,
+                             "anyof_type": ["date", "string"]},
+                "name": {"description": "name of meeting",
+                         "required": False,
+                         "type": "string"},
+                "objective": {"description": "goal of the meeting",
+                              "required": False,
+                              "type": "string"},
+                "audience": {"description": "rolls of audience members "
+                                            "e.g. 'pi', 'lead'",
+                             "required": False,
+                             "type": "list"},
+                "notes": {"description": "any notes about the kickoof",
+                          "required": False,
+                          "type": "list"},
+                "status": {"description": "proposed, started, finished, converged",
+                           "required": False,
+                           "type": "string",
+                           "eallowed": PROJECTA_STATUS}
+
+            }
+        },
+        "lead": {
+            "description": "id of projecta lead",
```

Comment:
"the id of the lead student or person for the projectum.  Person details will be dereferenced from the people collection."